src/like-service/app/Http/Controllers/DoLikeController.php

```diff
@@ -4,10 +4,39 @@
 use Illuminate\Foundation\Bus\DispatchesJobs;
 use Illuminate\Foundation\Validation\ValidatesRequests;
 use Illuminate\Routing\Controller as BaseController;
-public class DoLikeController extends BaseController{
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Log;
+
+class DoLikeController extends BaseController{
     use AuthorizesRequests, DispatchesJobs, ValidatesRequests;
 
     function doLike($request){
+        $user_token = $request->cookie('jwt');
+
+        if(!$this->authenticateToken($user_token)){
+            return response()->json([
+                'message' => 'Unauthorized'
+            ], 401);
+        }else{
+            //Todo: do stuff
+            Log::notice('Do stuff');
+        }
+
+    }
+
+    function authenticateToken($user_token){
```

Comment:
[DISCUSS] What do you think of renaming this to `validateToken`? This does not authenticate the token (it authenticates the user who provides the token)

Reply:
Renamed